sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -585,6 +585,18 @@ class SparkSession private(
     sparkContext.stop()
   }
 
+  /**
+   * Return whether the SparkSession is using hive catalog or not
+   * @return
+   * @since 2.0.0
+   */
+  def usingHive: Boolean = {
```

Comment:
I can remove it because SparkILooop will not use it to display whether we are using hive catalog.